keanu-python/keanu/vertex.py

```diff
@@ -109,8 +109,23 @@ def __init__(self, ctor, *args):
         super(Vertex, self).__init__(ctor, *(Vertex.__parse_args(*args)))
 
     def observe(self, v):
-        from keanu.tensor import Tensor
-        self.unwrap().observe(Tensor(v).unwrap())
+        self.unwrap().observe(kn.Tensor(v).unwrap())
+
+    def setValue(self, v):
```

Comment:
isn't the `python` way `set_value` etc?

Reply:
yes, sorry - force of habit!

Reply:
This is actually a method overload. If I renamed this to `set_value` then the user could still call `setValue` because it's handled by `JavaObjectWrapper`'s `__getattr__`.

In a later PR Joanne improves on this so that we convert snake case python calls to camel case Java calls automatically.